Doc/Tutorial/chapter_pdb.tex

```diff
@@ -93,24 +93,28 @@ \subsection{Reading an mmCIF file}
 \subsection{Reading files in the MMTF format}
 
 You can use the direct MMTFParser to read a structure from a file:
+%doctest ../Tests lib:mmtf
 \begin{verbatim}
 >>> from Bio.PDB.mmtf import MMTFParser
 >>> structure = MMTFParser.get_structure("PDB/4CUP.mmtf")
```

Comment:
Unfortunately this triggers a warning:

```
$ python3.5
Python 3.5.1 (default, Jun 23 2016, 12:06:32) 
[GCC 4.2.1 Compatible Apple LLVM 7.3.0 (clang-703.0.31)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from Bio.PDB.mmtf import MMTFParser
>>> structure = MMTFParser.get_structure("PDB/4CUP.mmtf")
/Users/xxx/lib/python3.5/site-packages/Bio/PDB/StructureBuilder.py:85: PDBConstructionWarning: WARNING: Chain A is discontinuous at line 0.
  PDBConstructionWarning)
```

Is there a better behaved MMTK file we could use for this test (a second small MMTK file)?